bundle/src/main/java/com/adobe/acs/commons/marketo/impl/MarketoClientConfigurationImpl.java

```diff
@@ -80,6 +80,9 @@ public String getClientSecret() {
 
   @Override
   public String getEndpointHost() {
+    if(endpointHost.startsWith("https://")){
+      return endpointHost.substring(8);
```

Comment:
```suggestion
      return endpointHost.substring("https://".length());
```
Just a but more clear than the magic numbe `8`